tests/test_reader.py

```diff
@@ -0,0 +1,54 @@
+import csv
+
+import pytest
+from mock import Mock
+
+from datarobot_batch_scoring.reader import FastReader, SlowReader
+
+
+class TestCSVReaderWithWideData(object):
```

Comment:
I used to think like this too, but organizing tests into test classes makes it easier to avoid shooting yourself in the foot with fixture names in the module.

Like, if you define a test fixture `foo` then forget to include it as a dependency of the script, your IDE won't be able to tell you that you've done something wrong because there is a `foo` object accessible in the namespace.

It's just for grouping of tests - there is no unittest style test setup or anything else like that. I promise it's an enhancement, not a regression.